CooperationHullMainSite/wwwroot/css/site.css

```diff
@@ -231,21 +231,34 @@ body {
 
 
 
-    /* Homepage block */
+/* Homepage block */
 
-.homepage-block > div {
-        min-height:10vh;
-        margin-left:0.1rem;
-        margin-right:0.1rem;
+.homepage-block > .row {
+    padding-top: 4rem;
+    padding-bottom: 4rem;
+    padding-left:5rem;
+    padding-right: 5rem;
+}
+
+@media screen and (max-width: 600px) {
+    .homepage-block > .row {
+        padding-left: 4rem;
+        padding-right: 4rem;
     }
+}
```

Comment:
nit: so far we're using `min-width` media queries so that we're designing mobile-first. Generally this is considered good pracice these days as most users are on mobile, and for consitency's sake with the rest of the codebase it might be worth doing here as well

Reply:
good shout thank you. Appreciate little tidbits like this as I'm not up to date on my front end dev